cart.html

```diff
@@ -0,0 +1,117 @@
+    <script src="https://kit.fontawesome.com/e4b1529bdb.js" crossorigin="anonymous"></script>
+    <link href="./cart.css" rel="stylesheet" />
+    <title>Nursery Mart | Cart</title>
+</head>
+
+<body>
+    <header>
+        <img class="nav-img" src="./assets/logo.PNG" />
+        <div class="search">
+            <input placeholder="Search Item" class="search-bar" />
+            <button class="btn-search">
+                <i class="fas fa-search"></i>
+            </button>
+        </div>
+        <div class="icons">
+            <button class="login btn primary-solid-btn">LOGIN</button>
+            <button class="icon-btn btn-no-decor">
+                <i class="far fa-2x fa-heart"></i>
+                <div class="badge-right badge-text badge-round">5</div>
+            </button>
+            <button class="icon-btn btn-no-decor">
+                <i class="fas fa-2x fa-shopping-cart"></i>
+                <div class="badge-right badge-text badge-round">6</div>
+            </button>
+        </div>
+    </header>
+    <main>
+        <aside class="cart-items">
+            <div class="horizontal-card">
+                <img class="img-card-horizontal" src="Assets/Indoor/4-succulents-indoor.jpg" alt="Socculent Plants" loading="lazy"/>
+                <div class="card-horizontal-desc">
+                    <p>Top 4 Die Succulent Indoor Plants Pack</p>
+                    <div class="horizontal-price"><span class="stroked-price"><i
+                                class="fas ruppee-stroke fa-rupee-sign"></i> 995</span>
+                        <span class="actual-price bold"><i class="fas ruppee fa-rupee-sign"></i> 749</span></div>
+                    <div class="disc-percent">25%</div>
+                    <div class="quantity-cart-item">Quantity:
+                        <button class="btn-no-decor"><i class="fas fa-minus-circle"></i></button>
+                        <input class="input-quantity" min="1" value="1" />
+                        <button class="btn-no-decor"><i class="fas fa-plus-circle"></i></button>
+                    </div>
+                    <button class="btn primary-solid-btn">Remove from Cart</button>
+                    <button class="btn secondary-solid-btn">Move to Wishlist</button>
+                </div>
+            </div>
+            <div class="horizontal-card">
+                <img class="img-card-horizontal" src="Assets/Outdoor/gladiolus-random-color.jpg" alt="Gladiolus flower" loading="lazy"/>
+                <div class="card-horizontal-desc">
+                    <p>Gladiolus flower bulbs (Random color)</p>
+                    <div class="horizontal-price"><span class="stroked-price"><i
+                                class="fas ruppee-stroke fa-rupee-sign"></i> 200</span>
+                        <span class="actual-price bold"><i class="fas ruppee fa-rupee-sign"></i> 189</span></div>
+                    <div class="disc-percent">10%</div>
```

Comment:
IMO, you can write like "10% off". Just "10%" looks confusing.

Reply:
Yeah, will change it